### DISCUSSION TITLE
mutateRelationshipDataBeforeFillUsing not work

### DISCUSSION THREAD

**Discussion by edson36** [filamentphp/filament#12066] [Help]
### Package

Panel builder

### Package Version

v3.0

### How can we help you?

How to show date in d/m/Y format without datepicker coming from a relationship? DatePicker solves the formatting, but users with visual difficulties find it difficult to use the picker,
that's why they don't want to use the datepicker, I tried using mutateRelationshipDataBeforeFillUsing but it doesn't work for relationship, mutateRelationshipDataBeforeSaveUsing works for saving, but mutateRelationshipDataBeforeFillUsing doesn't for filling
![Captura de Tela 2024-03-24 às 17 20 35](https://github.com/filamentphp/filament/assets/6059053/3ea217d5-ecaa-42da-a335-bc025ca306e4)
![Captura de Tela 2024-03-24 às 17 21 32](https://github.com/filamentphp/filament/assets/6059053/aae18df9-0e0b-40ef-a988-2f36b1eb6ddc)

**Comment by edson36** [comment]
Opa @leandrocfe gostaria de saber se posso usar (copiar) e adequar o ptbr-form-fields para fazer adequação da date e datetime para esta minha aplicação? estou parado nesta situação

**Comment by leandrocfe** [comment]
Qual o problema de usar o DateTimePicker?

**Reply by edson36** [reply]
os usuário tem dificuldades em usar o picker para selecionar a data, foi pedido a mim que fosse apenas um campo para digitar a data (nascimento) via teclas, então eu usei o mask, porém para converter a data (ENG -> PT-BR) antes de preencher ou (PT-BR -> ENG) para antes de salvar é preciso usar o mutate, até funciona para relationship porém quando há data acontece algum bug pq se eu fizer dd dentro da função mutate aparece a data convertida (o q seria o correto), porém ao preencher (fill) a data fica original do BD em ENG

**Reply by PegeDev** [reply]
> Qual o problema de usar o DateTimePicker?

I'm experiencing the same thing, seems like this is a bug..

**Comment by PegeDev** [comment]
any update for this issue ?
